man/journald.conf.xml

```diff
@@ -18,6 +18,7 @@
   <refnamediv>
     <refname>journald.conf</refname>
     <refname>journald.conf.d</refname>
+    <refname>journald@.conf</refname>
```

Comment:
What is `journald@.conf` vs `journald@<NAMESPACE>.conf`?  Is that the base template file that you build into a namespace configuration?

Reply:
that means nothing. It's just that the man page name is generated from what is specified here, and including `<NAMESPACE>` in the man page name makes it hard to type in the shell... Hence I figured we just write in in the template syntax systemd uses for units, even though it's a bit sloppy, since we don't do templating here...